### DISCUSSION TITLE
Systematic rescan issue of some plugins with latest beta

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#774] [Help]
Hi,

With the latest beta, everytime I start AG it rescans a lot of plugin (and everytime the same ones !).

This is really annoying as the scan is really slow, and AG can needs several dozens of minutes to start !!
What makes it worst is when AG crashes, I can be stuck for half an hour up to an hour until the rescan is done and the AG server is up again.

 I will post some full logs on that later. It was not the case with previous beta (up to beta 3 I think).

By the way I dont know how steinberg handle that, but their VST/VST3 plugin scan is incredibly faster than AG and some others (metaplugin is quite fast too, native is really slow like AG).

**Comment by CaBuffalo** [comment]
This is very strange. It looks like the directory setting problem to me.
Are your plugins at the default directories?

On Fri, Oct 22, 2021 at 1:38 AM Henri75 ***@***.***> wrote:

> Hi,
>
> With the latest beta, everytime I start AG it rescans a lot of plugin (and
> everytime the same ones !).
>
> This is really annoying as the scan is really slow, and AG can needs
> several dozens of minutes to start !!
> What makes it worst is when AG crashes, I can be stuck for half an hour up
> to an hour until the rescan is done and the AG server is up again.
>
> I will post some full logs on that later. It was not the case with
> previous beta (up to beta 3 I think).
>
> By the way I dont know how steinberg handle that, but their VST/VST3
> plugin scan is incredibly faster than AG and some others (metaplugin is
> quite fast too, native is really slow like AG).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/774>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AFUOVUK6N6OYMPOXDUIEPINANCNFSM5GP75U7A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Henri75** [comment]
A lot are indeed in additional directories (like AA stuff) and set in AG
settings.
But at least a few of the ones that are rescanned are in regular directory
(even if most of them seems to be in additional folders, but may be it is
because all acustica audio plugins are in additional folders)

**Comment by CaBuffalo** [comment]
Try to put those directories into the setting and see what happens.

On Sat, Oct 23, 2021 at 7:30 AM Henri75 ***@***.***> wrote:

> A lot are indeed in additional directories (like AA stuff) and set in AG
> settings.
> But at least a few of the ones that are rescanned are in regular directory
> (even if most of them seems to be in additional folders, but may be it is
> because all acustica audio plugins are in additional folders)
>
>
> Le ven. 22 oct. 2021 à 16:51, CaBuffalo ***@***.***> a écrit :
>
> > This is very strange. It looks like the directory setting problem to me.
> > Are your plugins at the default directories?
> >
> > On Fri, Oct 22, 2021 at 1:38 AM Henri75 ***@***.***> wrote:
> >
> > > Hi,
> > >
> > > With the latest beta, everytime I start AG it rescans a lot of plugin
> > (and
> > > everytime the same ones !).
> > >
> > > This is really annoying as the scan is really slow, and AG can needs
> > > several dozens of minutes to start !!
> > > What makes it worst is when AG crashes, I can be stuck for half an hour
> > up
> > > to an hour until the rescan is done and the AG server is up again.
> > >
> > > I will post some full logs on that later. It was not the case with
> > > previous beta (up to beta 3 I think).
> > >
> > > By the way I dont know how steinberg handle that, but their VST/VST3
> > > plugin scan is incredibly faster than AG and some others (metaplugin is
> > > quite fast too, native is really slow like AG).
> > >
> > > —
> > > You are receiving this because you are subscribed to this thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/apohl79/audiogridder/discussions/774>, or
> > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6AFUOVUK6N6OYMPOXDUIEPINANCNFSM5GP75U7A
> > >
> > > .
> > > Triage notifications on the go with GitHub Mobile for iOS
> > > <
> >
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> > >
> > > or Android
> > > <
> >
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> > >.
> > >
> > >
> >
> >
> > --
> > Kevin Ma
> > Supported by Gmail.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/apohl79/audiogridder/discussions/774#discussioncomment-1521757
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ANMF6UBGNUHBKKCT2WHDRQLUIF26RANCNFSM5GP75U7A
> >
> > .
> > Triage notifications on the go with GitHub Mobile for iOS
> > <
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> >
> > or Android
> > <
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >.
> >
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/774#discussioncomment-1525211>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BTQCBXPSMXZZ3JAKLUILBHHANCNFSM5GP75U7A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Henri75** [comment]
All additional directories outside standard path are already added in
settings.

I noticed too that à lot of plugins that works in cubase, maschine and
studio one are blacklisted.
Moreover when i unblock them and they are rescan à few of them are finally
validated. Which means with à lot of unblock and restart i got à lof of
them to be finally available...
Then there is those than will never be validated.  A lof of them (may be
half) are nebula setups and they works fine in daw.
Really weird

**Comment by CaBuffalo** [comment]
Did you check off "Disable server filer" in the plugin menu of plugin
window?

On Sat, Oct 23, 2021 at 1:44 PM Henri75 ***@***.***> wrote:

> All additional directories outside standard path are already added in
> settings.
>
> I noticed too that à lot of plugins that works in cubase, maschine and
> studio one are blacklisted.
> Moreover when i unblock them and they are rescan à few of them are finally
> validated. Which means with à lot of unblock and restart i got à lof of
> them to be finally available...
> Then there is those than will never be validated. A lof of them (may be
> half) are nebula setups and they works fine in daw.
> Really weird
>
>
>
> On Sat, 23 Oct 2021, 18:38 CaBuffalo, ***@***.***> wrote:
>
> > Try to put those directories into the setting and see what happens.
> >
> > On Sat, Oct 23, 2021 at 7:30 AM Henri75 ***@***.***> wrote:
> >
> > > A lot are indeed in additional directories (like AA stuff) and set in
> AG
> > > settings.
> > > But at least a few of the ones that are rescanned are in regular
> > directory
> > > (even if most of them seems to be in additional folders, but may be it
> is
> > > because all acustica audio plugins are in additional folders)
> > >
> > >
> > > Le ven. 22 oct. 2021 à 16:51, CaBuffalo ***@***.***> a écrit :
> > >
> > > > This is very strange. It looks like the directory setting problem to
> > me.
> > > > Are your plugins at the default directories?
> > > >
> > > > On Fri, Oct 22, 2021 at 1:38 AM Henri75 ***@***.***> wrote:
> > > >
> > > > > Hi,
> > > > >
> > > > > With the latest beta, everytime I start AG it rescans a lot of
> plugin
> > > > (and
> > > > > everytime the same ones !).
> > > > >
> > > > > This is really annoying as the scan is really slow, and AG can
> needs
> > > > > several dozens of minutes to start !!
> > > > > What makes it worst is when AG crashes, I can be stuck for half an
> > hour
> > > > up
> > > > > to an hour until the rescan is done and the AG server is up again.
> > > > >
> > > > > I will post some full logs on that later. It was not the case with
> > > > > previous beta (up to beta 3 I think).
> > > > >
> > > > > By the way I dont know how steinberg handle that, but their
> VST/VST3
> > > > > plugin scan is incredibly faster than AG and some others
> (metaplugin
> > is
> > > > > quite fast too, native is really slow like AG).
> > > > >
> > > > > —
> > > > > You are receiving this because you are subscribed to this thread.
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/apohl79/audiogridder/discussions/774>, or
> > > > unsubscribe
> > > > > <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6AFUOVUK6N6OYMPOXDUIEPINANCNFSM5GP75U7A
> > > > >
> > > > > .
> > > > > Triage notifications on the go with GitHub Mobile for iOS
> > > > > <
> > > >
> > >
> >
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> > > > >
> > > > > or Android
> > > > > <
> > > >
> > >
> >
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> > > > >.
> > > > >
> > > > >
> > > >
> > > >
> > > > --
> > > > Kevin Ma
> > > > Supported by Gmail.
> > > >
> > > > —
> > > > You are receiving this because you authored the thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/apohl79/audiogridder/discussions/774#discussioncomment-1521757
> > > >,
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ANMF6UBGNUHBKKCT2WHDRQLUIF26RANCNFSM5GP75U7A
> > > >
> > > > .
> > > > Triage notifications on the go with GitHub Mobile for iOS
> > > > <
> > >
> >
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> > > >
> > > > or Android
> > > > <
> > >
> >
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> > > >.
> > > >
> > > >
> > >
> > > —
> > > You are receiving this because you commented.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/apohl79/audiogridder/discussions/774#discussioncomment-1525211
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ARMAR6BTQCBXPSMXZZ3JAKLUILBHHANCNFSM5GP75U7A
> > >
> > > .
> > > Triage notifications on the go with GitHub Mobile for iOS
> > > <
> >
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> > >
> > > or Android
> > > <
> >
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> > >.
> > >
> > >
> >
> >
> > --
> > Kevin Ma
> > Supported by Gmail.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/apohl79/audiogridder/discussions/774#discussioncomment-1525556
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ANMF6UDYR4ZV27KOCC4OXVDUILQHZANCNFSM5GP75U7A
> >
> > .
> > Triage notifications on the go with GitHub Mobile for iOS
> > <
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> >
> > or Android
> > <
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >.
> >
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/774#discussioncomment-1526097>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6G2QMUECHYVZVVQ5SDUIMNDFANCNFSM5GP75U7A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Henri75** [comment]
I didnt touch it. What is the use ?

There is something weird because it is rescanning plugin that were previously validated .

I checked AG logs today, here is an example of one ot the hundreds of plugins rescanned every time (which means up to 1 hour after I boot my mac for AG to ready) :

-rw-r--r--     1 nasta  staff   1.1K Nov  5 06:38 Users-nasta-Library-Audio-Plug-Ins-VST3-HoRNetVUMeterMK4-vst3-VST3_2021-11-05_06-38-56.log
-rw-r--r--     1 nasta  staff   1.1K Nov  5 14:45 Users-nasta-Library-Audio-Plug-Ins-VST3-HoRNetVUMeterMK4-vst3-VST3_2021-11-05_14-45-57.log
-rw-r--r--     1 nasta  staff   1.1K Nov  6 07:36 Users-nasta-Library-Audio-Plug-Ins-VST3-HoRNetVUMeterMK4-vst3-VST3_2021-11-06_07-36-16.log
-rw-r--r--     1 nasta  staff   1.1K Nov  7 08:30 Users-nasta-Library-Audio-Plug-Ins-VST3-HoRNetVUMeterMK4-vst3-VST3_2021-11-07_08-30-19.log
-rw-r--r--     1 nasta  staff   1.1K Nov  7 11:42 Users-nasta-Library-Audio-Plug-Ins-VST3-HoRNetVUMeterMK4-vst3-VST3_2021-11-07_11-42-03.log
-rw-r--r--     1 nasta  staff   1.1K Nov  8 08:18 Users-nasta-Library-Audio-Plug-Ins-VST3-HoRNetVUMeterMK4-vst3-VST3_2021-11-08_08-18-26.log

or 

-rw-r--r--     1 nasta  staff   1.1K Nov  2 15:38 Library-Audio-Plug-Ins-VST3-AcusticaVST3-Setups-Teletronix-LA2A-3A-vst3-VST3_2021-11-02_15-38-20.log
-rw-r--r--     1 nasta  staff   1.1K Nov  3 20:38 Library-Audio-Plug-Ins-VST3-AcusticaVST3-Setups-Teletronix-LA2A-3A-vst3-VST3_2021-11-03_20-38-34.log
-rw-r--r--     1 nasta  staff   1.1K Nov  4 19:40 Library-Audio-Plug-Ins-VST3-AcusticaVST3-Setups-Teletronix-LA2A-3A-vst3-VST3_2021-11-04_19-40-28.log
-rw-r--r--     1 nasta  staff   1.1K Nov  5 14:32 Library-Audio-Plug-Ins-VST3-AcusticaVST3-Setups-Teletronix-LA2A-3A-vst3-VST3_2021-11-05_14-32-20.log
-rw-r--r--     1 nasta  staff   1.1K Nov  6 07:23 Library-Audio-Plug-Ins-VST3-AcusticaVST3-Setups-Teletronix-LA2A-3A-vst3-VST3_2021-11-06_07-23-11.log
-rw-r--r--     1 nasta  staff   1.1K Nov  7 11:26 Library-Audio-Plug-Ins-VST3-AcusticaVST3-Setups-Teletronix-LA2A-3A-vst3-VST3_2021-11-07_11-26-38.log



In the scan logs it is validated 

[08:18:26.565|app|7f8447822ca8] commandline: -scan /Users/nasta/Library/Audio/Plug-Ins/VST3/HoRNetVUMeterMK4.vst3|VST3
[08:18:26.565|app|7f8447822ca8] scan mode: format=VST3 id=/Users/nasta/Library/Audio/Plug-Ins/VST3/HoRNetVUMeterMK4.vst3
[08:18:26.565|server|1099b3af0] scanning id=/Users/nasta/Library/Audio/Plug-Ins/VST3/HoRNetVUMeterMK4.vst3 fmt=VST3
[08:18:28.278|server|1099b3af0] adding plugin description:
[08:18:28.278|server|1099b3af0]   name            = HoRNetVUMeterMK4 (HoRNetVUMeterMK4)
[08:18:28.278|server|1099b3af0]   uniqueId        = ae4e22e1
[08:18:28.278|server|1099b3af0]   deprecatedUid   = a607f55
[08:18:28.278|server|1099b3af0]   id string       = VST3-HoRNetVUMeterMK4-a607f55
[08:18:28.278|server|1099b3af0]   manufacturer    = HoRNet
[08:18:28.278|server|1099b3af0]   category        = Fx|Analyzer
[08:18:28.278|server|1099b3af0]   shell           = 0
[08:18:28.278|server|1099b3af0]   instrument      = 0
[08:18:28.278|server|1099b3af0]   input channels  = 2
[08:18:28.278|server|1099b3af0]   output channels = 2
[08:18:28.480|app|7f8447822ca8] ...success
[08:18:28.480|app|7f8447822ca8] initialise complete

**Comment by Pete3030** [comment]
I have the same problem with latest Beta on Mac High Sierra..
